front/app/modules/commercial/insights/admin/containers/Insights/Details/index.tsx

```diff
@@ -21,16 +38,136 @@ const Container = styled.div`
 `}
 `;
 
-const DetailsInsightsView = () => {
+const Left = styled.div`
+  position: relative;
+  width: 100%;
+`;
+
+const DetailsInsightsView = ({
+  params: { viewId },
+  location: { pathname, query },
+}: WithRouterProps) => {
+  const [isPreviewOpen, setIsPreviewOpen] = useState(false);
+  const [previewedInputIndex, setPreviewedInputIndex] = useState<number | null>(
+    null
+  );
+  // Use ref for isPreviewedInputInTable to avoid dependencies in moveUp and moveDown
+  const isPreviewedInputInTable = useRef(true);
```

Comment:
Can the previewed input disappear from table in the details page ? I was under the impression it couldn't and if it's true maybe we want to simplify the logic and remove this part ?

Reply:
Yes, I was also thinking about this but I wasn't sure. Now I confirmed with Karel and indeed the logic can be simplified. It's done now. Another reason to copy/paste the navigation logic as it's already diverging ;) Still, agree that we should keep an eye out for possible useful abstractions.